doc/kubernetes/modules/ROOT/pages/running/timeout_tunning.adoc

```diff
@@ -0,0 +1,103 @@
+= Understanding failure detection in Infinispan and JGroups
+:description: This guide describes how JGroups detects failed instances and its configuration.
+
+{description}
+
+== Audience
+
+This guide describe the failure detection mechanism that JGroups enforces to detect crashed pods inside the Infinispan cluster.
+It explains the configuration attributes available for customization and its trade-off to achieve the desired detection rate.
+
+== Broken TCP connections
```

Comment:
@belaban when you get a chance, can you take a look at this section

Reply:
* "have a performance impact" -> "has a ..."
* "estabilished"
* "The timeouts are tuning..."?

Reply:
LGTM!

Reply:
Thanks for the comments!